.travis.yml

```diff
@@ -20,8 +20,11 @@ matrix:
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*@dev"
+    - php: 5.6
+      env: SYMFONY_VERSION="3.3.*@dev"
   allow_failures:
     - php: nightly
+    - env: SYMFONY_VERSION="3.3.*@dev"
```

Comment:
don't enforce ``3.3.*@dev``, which will force us to maintain the list every time.

Instead, add a job changing the min stability to test against dev version of all vendors. See https://github.com/minkphp/Mink/blob/9ea1cebe3dc529ba3861d87c818f045362c40484/.travis.yml to see how to do it